example/src/App.tsx

```diff
@@ -115,8 +108,17 @@ const App = () => {
           name: "Dark",
           keywords: "dark theme",
           section: "",
-          perform: () =>
-            document.documentElement.setAttribute("data-theme-dark", ""),
+          perform: () => {
+            document.documentElement.setAttribute("data-theme-dark", "");
+            return () => {
```

Comment:
Return an optional cleanup function similar to `useEffect` which will give your action the ability to undo.